spp_base_gis/models/gis_test_model.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class GisTestModel(models.Model):
+    _name = "spp.base.gis.test.model"
+    _description = "GIS Test Model"
+
+    name = fields.Char()
+    geo_polygon_field = fields.GeoPolygonField()
+    geo_line = fields.GeoLineStringField()
+    geo_point = fields.GeoPointField()
```

Comment:
It is better to move the test models into test projects. That way we do not install the test models in prod. Move the tests to this test project.